src/env.h

```diff
@@ -174,6 +177,7 @@ namespace node {
   V(replacement_string, "replacement")                                        \
   V(retry_string, "retry")                                                    \
   V(rss_string, "rss")                                                        \
+  V(run_main_string, "_runMain")                                              \
```

Comment:
If you are using the strings only once, you can just create them inline, no need to create persistent handles for them.  This is really only needed for strings that are created frequently.